test/00-git-config.t

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+source test/setup
+
+use Test::More
+
+note "Define project-wide GIT setup for all tests"
+
+# Get git-subrepo project top directory
+PROJ_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
+
+if [ -z "${PROJ_DIR}" ] || [ "${HOME}" != "${PROJ_DIR}" ]; then
+  is "${HOME}" "${PROJ_DIR}" \
+  "To define project-wide GIT setup for all tests: HOME '${HOME}' should equal PROJ_DIR '${PROJ_DIR}'"
+else
+
+  # Real GIT configuration for tests is set here:
+  rm -f "${PROJ_DIR}/.gitconfig"
```

Comment:
I am not ok with removing the .gitconfig from a person's working directory.

Reply:
Hi again,
i thought i made sure, that HOME is temporarily set to $PWD in 'test/setup' for the duration of running tests and only when git-subrepo's clone directory (PROJ_DIR) and HOME match, then rewrite tests-defined git configuration in PROJ_DIR.
regards, Samo